runtime/port/win32/j9shmem.c

```diff
@@ -886,26 +885,24 @@ j9shmem_createDir(struct J9PortLibrary* portLibrary, char* cacheDirName, uintptr
 	Trc_PRT_j9shmem_createDir_ExitFailed();
 	return -1;
 }
- 
+
 intptr_t
-j9shmem_getFilepath(struct J9PortLibrary* portLibrary, char* cacheDirName, char* buffer, uintptr_t length, const char* cachename)
+j9shmem_getFilepath(struct J9PortLibrary *portLibrary, char *cacheDirName, char *buffer, uintptr_t length, const char *cacheName)
 {
 	OMRPORT_ACCESS_FROM_J9PORT(portLibrary);
- 	
- 	char *filePath;
-	
-	if (cacheDirName == NULL) {
+	char *filePath = NULL;
+	if (NULL == cacheDirName) {
```

Comment:
I think the blank line between the declarations and the code (i.e. line 896) was clarifying: Could you please restore it?